pkg/actions/project.go

```diff
@@ -284,3 +284,155 @@ func ProjectRestart(c *cli.Context) {
 	fmt.Println(string(response))
 	os.Exit(0)
 }
+
+// ProjectLinkList : lists all the links for a project
+func ProjectLinkList(c *cli.Context) {
+	projectID := strings.TrimSpace(strings.ToLower(c.String("id")))
+
+	conID, getConnectionIDErr := project.GetConnectionID(projectID)
+	if getConnectionIDErr != nil {
+		HandleProjectError(getConnectionIDErr)
+		os.Exit(1)
+	}
+
+	conInfo, conInfoErr := connections.GetConnectionByID(conID)
+	if conInfoErr != nil {
+		HandleConnectionError(conInfoErr)
+		os.Exit(1)
+	}
+
+	conURL, conErr := config.PFEOriginFromConnection(conInfo)
+	if conErr != nil {
+		HandleConfigError(conErr)
+		os.Exit(1)
+	}
+
+	links, projectLinkErr := project.GetProjectLinks(http.DefaultClient, conInfo, conURL, projectID)
+	if projectLinkErr != nil {
+		HandleProjectError(projectLinkErr)
+		os.Exit(1)
+	}
+
+	if printAsJSON {
+		json, _ := json.Marshal(links)
+		fmt.Println(string(json))
+	} else {
+		if len(links) == 0 {
+			fmt.Println("No projects bound to Codewind")
+		} else {
+			w := new(tabwriter.Writer)
+			w.Init(os.Stdout, 0, 8, 2, '\t', 0)
+			fmt.Fprintln(w, "ENVIRONMENT VARIABLE \tPROJECT ID")
+			for _, project := range links {
+				fmt.Fprintln(w, project.EnvName+"\t"+project.ProjectID)
+			}
+			fmt.Fprintln(w)
+			w.Flush()
+		}
+	}
+	os.Exit(0)
+}
+
+// ProjectLinkCreate : creates a new link
+func ProjectLinkCreate(c *cli.Context) {
+	projectID := strings.TrimSpace(strings.ToLower(c.String("id")))
+	targetProjectID := strings.TrimSpace(strings.ToLower(c.String("targetID")))
+	envName := strings.TrimSpace(c.String("env"))
+
+	conID, getConnectionIDErr := project.GetConnectionID(projectID)
+	if getConnectionIDErr != nil {
+		HandleProjectError(getConnectionIDErr)
+		os.Exit(1)
+	}
+
+	conInfo, conInfoErr := connections.GetConnectionByID(conID)
+	if conInfoErr != nil {
+		HandleConnectionError(conInfoErr)
+		os.Exit(1)
+	}
+
+	conURL, conErr := config.PFEOriginFromConnection(conInfo)
+	if conErr != nil {
+		HandleConfigError(conErr)
+		os.Exit(1)
+	}
+
+	projectLinkErr := project.CreateProjectLink(http.DefaultClient, conInfo, conURL, projectID, targetProjectID, envName)
+	if projectLinkErr != nil {
+		HandleProjectError(projectLinkErr)
+		os.Exit(1)
+	}
+
+	response, _ := json.Marshal(project.Result{Status: "OK", StatusMessage: "Project link create request accepted"})
```

Comment:
do you need to consider `--json` here (if it's not set)?

Reply:
I've followed the general layout of the commands in `project.go` and most just print the response